Sources/TestsCommon/CommonTestUtils.swift

```diff
@@ -30,33 +39,28 @@ open class MongoSwiftTestCase: XCTestCase {
     /// Gets the connection string to use from the environment variable, $MONGODB_URI. If the variable does not exist,
     /// will return a default of "mongodb://127.0.0.1/". If singleMongos is true and this is a sharded topology, will
     /// edit $MONGODB_URI as needed so that it only contains a single host.
-    public static func getConnectionString(singleMongos: Bool = true) -> String {
-        let uri = Self.uri
-
+    public static func getConnectionString(singleMongos: Bool = true) -> ConnectionString {
```

Comment:
I updated these helpers to operate over our `ConnectionString` type rather than `String` since it has a lot of useful utilities.